pkg/chain/local/ids.go

```diff
@@ -0,0 +1,61 @@
+package local
+
+import (
+	cecdsa "crypto/ecdsa"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
```

Comment:
This import causes the fail of the Celo CI job. `go-ethereum/crypto` conflicts with `celo-blockchain/crypto` package used by the Celo code (when `celo` build tag is used). We need to extract `crypto.PubkeyToAddress` to separate files and use a common alias. Those files can be, for example, `crypto_ethereum.go` which will import `go-ethereum/crypto`  and `crypto_celo.go` which will take `celo-blockchain/crypto`. Both files should also use correct build tags.

Reply:
Fixed in https://github.com/keep-network/keep-ecdsa/pull/744/commits/b780e8f2ea2b31184bff7e2331e3d0f1ecf71028

Reply:
I don't love `keep-core/pkg/operator` and the fact we need to call it hurts me but it is not blocking this PR. We will need to do something about `keep-core/pkg/operator` in the future though.